netpyne/sim/save.py

```diff
@@ -364,130 +364,59 @@ def intervalSave(simTime, gatherLFP=True):
     
     singleNodeVecs = ['t']
     simDataVecs = ['spkt','spkid','stims']+list(sim.cfg.recordTraces.keys())
-    if sim.nhosts > 1:
 
-        netPopsCellGids = {popLabel: list(pop.cellGids) for popLabel,pop in sim.net.pops.items()}
+    netPopsCellGids = {popLabel: list(pop.cellGids) for popLabel,pop in sim.net.pops.items()}
 
-        nodeData = {'netCells': [c.__getstate__() for c in sim.net.cells], 'netPopsCellGids': netPopsCellGids, 'simData': sim.simData}
-        data = [None]*sim.nhosts
-        data[0] = {}
-        for k,v in nodeData.items():
-            data[0][k] = v
+    nodeData = {'netCells': [c.__getstate__() for c in sim.net.cells], 'netPopsCellGids': netPopsCellGids, 'simData': sim.simData}
 
-        #print data
```

Comment:
This code is unchanged, only indent was changed due to it's no longer under `if sim.nhosts > 1`